GoogleForCreators/web-stories-wp
# Workspace: Show text sets for used fonts in story

## Summary

Show text sets that map to currently used fonts.

![image](https://user-images.githubusercontent.com/1738349/96178315-d8f9f100-0ef4-11eb-9b2b-29382b7b127d.png)

No text boxes with `UnifrakturMaguntia` 

![image](https://user-images.githubusercontent.com/1738349/96178344-e44d1c80-0ef4-11eb-8551-f8e697c48875.png)

## Relevant Technical Choices

Uses existing text set filter pattern to show text sets that map to currently used fonts.

## To-do

None

## User-facing changes

None (because feature flag)

Shows a new "Fonts In Use" pill that will show the correct text sets

## Testing Instructions

1. Create a story and add some text boxes.
2. `Open Sans` has a number of text sets and `UnifrakturMaguntia` is pretty unique. 
3. Delete one of the text boxes and see the text sets update.

---



#3640
